kfc/kfc-gradle-plugin/src/main/kotlin/com/github/turansky/kfc/gradle/plugin/PatchWebpackConfig.kt

```diff
@@ -108,10 +123,40 @@ open class PatchWebpackConfig : DefaultTask() {
             .asFile
 }
 
+@Suppress("JSUnnecessarySemicolon")
+private fun createReplacePatch(replacements: Map<String, Tuple2<String, Boolean>>): String? {
+    if (replacements.isEmpty())
+        return null
+
+    val replacementOptions = replacements.map { (oldValue, entry) ->
+        val (newValue, strict) = entry
+        "{ search: '$oldValue', replace: '$newValue', strict: $strict },"
+    }.joinToString("\n                ")
+
+    // language=JavaScript
+    return createPatch(
+        name = "string replacements",
+        body = """
+        config.module.rules.push(
+          {
+            test: /\.js$/,
+            loader: 'string-replace-loader',
+            options: {
+              multiple: [
```

Comment:
Did you test it with multiple dynamic imports?

Reply:
Multiple dynamic imports - very expensive and has no sense
Dynamic import pill is temp